make.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+set -e
+
+# Script inspired by https://gist.github.com/szeidner/613fe4652fc86f083cefa21879d5522b
+
+PROGNAME=$(basename $0)
+WORKING_DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+die() {
+    echo "$PROGNAME: $*" >&2
+    exit 1
+}
+
+usage() {
+    if [ "$*" != "" ] ; then
+        echo "Error: $*"
+    fi
+
+    cat << EOF
+Usage: $PROGNAME --bundle-id [BUNDLE_ID_PRODUCTION] --bundle-id-staging [BUNDLE_ID_STAGING] --app-name [PROJECT_NAME]
+Set up an iOS app from tuist template.
+Options:
+-h, --help                         display this usage message and exit
+-b, --bundle-id [BUNDLE_ID_PRODUCTION]  new production id (i.e. com.example.package)
+-s, --bundle-id-staging [BUNDLE_ID_STAGING]  new staging id (i.e. com.example.package.staging)
+-n, --app-name [PROJECT_NAME]          new app name (i.e. MyApp, "My App")
```

Comment:
Need to format a little bit here
```suggestion
-h, --help                                   display this usage message and exit
-b, --bundle-id [BUNDLE_ID_PRODUCTION]       the production id (i.e. com.example.package)
-s, --bundle-id-staging [BUNDLE_ID_STAGING]  the staging id (i.e. com.example.package.staging)
-n, --project-name [PROJECT_NAME]            the project name (i.e. MyApp)
```